joplin/js/editor.js

```diff
@@ -160,83 +165,71 @@ $(function() {
     return janisPreviewUrl;
   }
 
+  function setTabToContent(){
+    $('.tab-nav a').first().tab('show');
+    window.history.replaceState(null, null, $('.tab-nav a').first().attr('href'));
+  }
   // Changes language and update janisPreviewUrl for our language
   function changeLanguage(currentLang) {
     state.currentLang = currentLang;
+    setTabToContent();
 
-    var languageStrings = {
-      en: '[EN]',
-      es: '[ES]',
-      vi: '[VI]',
-      ar: '[AR]',
-    };
-
-    var lowerLanguageStrings = {
-      en: '[en]',
-      es: '[es]',
-      vi: '[vi]',
-      ar: '[ar]',
-    };
-
-    var languageRegex = /\[\w+\]/g;
-
-    // Hide stuff that isn't our language
-    // This is hacky but it seems to be working
-
-    // Top level fields
-    document.querySelectorAll('.object').forEach(elem => {
-      if (elem.querySelectorAll('.title-wrapper').length) {
-        var headerText = elem.querySelectorAll('.title-wrapper')[0].innerText;
-        var langString = headerText.match(languageRegex);
-        if (
-          langString != null &&
-          langString != languageStrings[currentLang] &&
-          langString != lowerLanguageStrings[currentLang]
-        ) {
-          elem.classList.add('hidden');
-        } else {
-          elem.classList.remove('hidden');
-        }
-      }
-    });
-
-    // Fields inside of InlinePanels
-    document.querySelectorAll('.field').forEach(elem => {
-      if (elem.querySelectorAll('label').length) {
-        var labelText = elem.querySelectorAll('label')[0].innerText;
-        var langString = labelText.match(languageRegex);
-
-        if (
-          langString != null &&
-          langString != lowerLanguageStrings[currentLang]
-        ) {
-          elem.parentElement.classList.add('hidden');
+    // replace brackets with hidden span tags
+    function replaceLanguageLabels() {
```

Comment:
This is https://github.com/cityofaustin/techstack/issues/2930, need to make sure that passes QA.